specification/flag-evaluation.md

```diff
@@ -124,7 +111,24 @@ MyStruct myStruct = client.getObjectValue<MyStruct>('structured-flag', { text: '
 
 See [evaluation context](./evaluation-context.md) for details.
 
-###### Conditional Requirement 1.3.3
+##### Condition 1.3.2
+
+> The implementation language differentiates between floating-point numbers and integers.
+
+###### Conditional Requirement 1.3.2.1
+
+> The client **SHOULD** provide functions for strings, numbers, booleans and structures, consistent with language idioms.
```

Comment:
I'm thinking this line should only include numbers because the condition above says "assuming your language differentiates between floating and non floating numbers.."?

Reply:
Oh good catch I've updated it.